kitsune/sumo/jinja2/includes/common_macros.html

```diff
@@ -131,16 +131,18 @@
   <div class="announce-bar {{ level }}" id="announce-{{ id }}">
     <div class="container_12">
       <div class="grid_12">
-        <div class="close-button" data-close-id="announce-{{ id }}"></div>
-        {{ caller() }}
+          <div class="close-button" data-close-id="announce-{{ id }}"></div>
```

Comment:
Extra spaces here a mistake? Not a problem, just odd as the old way looked better.

Reply:
Updated, it was my editor doing it's thing